qutebrowser/config/configdata.yml

```diff
@@ -954,6 +954,69 @@ hints.scatter:
 
     Ignored for number hints.
 
+hints.selectors:
+  default:
+    all:
+      - 'a'
+      - 'area'
+      - 'textarea'
+      - 'select'
+      - 'input:not([type="hidden"])'
+      - 'button'
+      - 'frame'
+      - 'iframe'
+      - 'img'
+      - 'link'
+      - 'summary'
+      - '[onclick]'
+      - '[onmousedown]'
+      - '[role="link"]'
+      - '[role="option"]'
+      - '[role="button"]'
+      - '[ng-click]'
+      - '[ngClick]'
+      - '[data-ng-click]'
+      - '[x-ng-click]'
+    links:
+      - 'a[href]'
+      - 'area[href]'
+      - 'link[href]'
+      - '[role="link"][href]'
+    images:
+      - 'img'
+    media:
+      - 'audio'
+      - 'img'
+      - 'video'
+    urls:
+      - '[src]'
+      - '[href]'
+    inputs:
+      - 'input[type="text"]'
+      - 'input[type="date"]'
+      - 'input[type="datetime-local"]'
+      - 'input[type="email"]'
+      - 'input[type="month"]'
+      - 'input[type="number"]'
+      - 'input[type="password"]'
+      - 'input[type="search"]'
+      - 'input[type="tel"]'
+      - 'input[type="time"]'
+      - 'input[type="url"]'
+      - 'input[type="week"]'
+      - 'input:not([type])'
+      - 'textarea'
+  type:
+    name: Dict
+    keytype: String
+    valtype:
+      name: List
+      none_ok: true
+      valtype: String
+  supports_pattern: true
+  desc: CSS selectors used to determine which elements on a page should have
```

Comment:
I think that the best way to solve the "users might override hints.selectors and miss out on updates" issue is to add documentation discouraging people from setting these values in autoconfig.yml. We could disable configuring this variable with `:set` too altogether as well.

Either way, I think this setting needs more description here than most, because it's very easy to mess up configuring this.